Library/Homebrew/env_config.rb

```diff
@@ -257,6 +257,10 @@ module EnvConfig
                      "`HOMEBREW_CLEANUP_PERIODIC_FULL_DAYS` days.",
         boolean:     true,
       },
+      HOMEBREW_NO_INSTALL_UPGRADE:            {
+        description: "If set, `brew install` will not automatically upgrade installed and out of date formulae",
```

Comment:
```suggestion
        description: "If set, `brew install` will not automatically upgrade installed and outdated formulae",
```

Reply:
How about `installed but outdated` rather than `installed and outdated`? Seems a bit clearer to be.

Reply:
Agree. Had this same thought when I wrote the PR title, but didn't remember it here. Thanks!